renku/core/commands/workflow.py

```diff
@@ -87,34 +86,123 @@ def rename_workflow_command():
 
 
 @inject.autoparams()
-def _remove_workflow(name, client: LocalClient):
-    """Remove the remote named <name>."""
-    from renku.core.models.refs import LinkReference
+def _remove_workflow(name, force, database: Database):
+    """Remove the workflow named <name>."""
+    now = datetime.utcnow()
+    workflows = unique_workflow(database)
+    not_found_text = f'The specified workflow is "{name}" is not an active workflow.'
```

Comment:
Should be declare closer to the exception that is raised (on the line above it or in the `ParameterError(...)` itself